docs/baremetalhost-states.md

```diff
@@ -45,9 +45,15 @@ profile.
 
 A host in the Ready state is available to be provisioned.
 
+## Preparing
+
+While setting the configuration of capabilities for the host,
```

Comment:
Maybe more details? At least mention RAID?

Reply:
Thanks, done.

Reply:
Maybe could be worth to mention that the Preparing state is required to perform _manual_ cleaning (as RAID cleaning step is not part of the _automated_ cleaning, performed later during the provisioning)?